### ISSUE TITLE
Where is to download dome135's '/media/posefs0c/panopticdb/a4/hdImgs/' data?

### ISSUE THREAD

**Issue by MasonForFun** [CMU-Perceptual-Computing-Lab/openpose_train#4]
Hi,

Thanks for the awsome project. I am training whole-body model by setting  sAddDome 1, but it seems that I need the following data. Can you tell me where I can get it?
sMediaDirectory = '/media/posefs0c/panopticdb/a4/hdImgs/'

### Posting rules
1. **Duplicated posts will not be answered**. Check the [FAQ](https://github.com/CMU-Perceptual-Computing-Lab/openpose/blob/master/doc/faq.md) section, other GitHub issues, and general documentation before posting. E.g., **low-speed, out-of-memory, output format, 0-people detected, installation issues, ...**).
2. **Fill** the **Your System Configuration section (all of it or it will not be answered!)** if you are facing an error or unexpected behavior. Feature requests or some other type of posts might not require it.
3. **No questions about 3rd party libraries**:
    - Caffe errors/issues, check [Caffe](http://caffe.berkeleyvision.org) documentation.
    - CUDA check failed errors: They are usually fixed by re-installing CUDA, then re-installing the proper cuDNN version, and then re-compiling (or re-installing) OpenPose. Otherwise, check for help in CUDA forums.
    - OpenCV errors: Install the default/pre-compiled OpenCV or check for online help.
4. Set a **proper issue title**: add the Ubuntu/Windows keyword and be specific (e.g., do not call it: `Error`).
5. Only English comments.
Posts which do not follow these rules will be **ignored, closed, or reported** with no further clarification.

### Executed Command (if any)
Note: add `--logging_level 0 --disable_multi_thread` to get higher debug information.

### Type of Issue
You might select multiple topics, delete the rest:
- Compilation/installation error
- Execution error
- Help wanted
- Question
- Enhancement / offering possible extensions / pull request / etc
- Other (type your own type)

### Your System Configuration
1. **Whole console output** (if errors appeared), paste the error to [PasteBin](https://pastebin.com/) and then paste the link here: LINK

2. **OpenPose version**: Latest GitHub code? Or specific commit (e.g., d52878f)? Or specific version from `Release` section (e.g., 1.2.0)?

3. **General configuration**:
    - **Installation mode**: CMake, sh script, manual Makefile installation, ... (Ubuntu); CMake, ... (Windows); ...?
    - **Operating system** (`lsb_release -a` in Ubuntu):
    - **Operating system version** (e.g., Ubuntu 16, Windows 10, ...):
    - **Release or Debug mode**? (by default: release):
    - Compiler (`gcc --version` in Ubuntu or VS version in Windows): 5.4.0, ... (Ubuntu); VS2015 Enterprise Update 3, VS2017 community, ... (Windows); ...?

4. **Non-default settings**:
    - **3-D Reconstruction module added**? (by default: no):
    - Any other custom CMake configuration with respect to the default version? (by default: no):

5. **3rd-party software**:
    - **Caffe version**: Default from OpenPose, custom version, ...?
    - **CMake version** (`cmake --version` in Ubuntu):
    - **OpenCV version**: pre-compiled `apt-get install libopencv-dev` (only Ubuntu); OpenPose default (only Windows); compiled from source? If so, 2.4.9, 2.4.12, 3.1, 3.2?; ...?

6. If **GPU mode** issue:
    - **CUDA version** (`cat /usr/local/cuda/version.txt` in most cases):
    - **cuDNN version**:
    - **GPU model** (`nvidia-smi` in Ubuntu):

7. If **CPU-only mode** issue:
    - **CPU brand & model**:
    - Total **RAM memory** available:

8. If **Python** API:
    - **Python version**: 2.7, 3.7, ...?
    - **Numpy version** (`python -c "import numpy; print numpy.version.version"` in Ubuntu):

9. If **Windows** system:
    - Portable demo or compiled library?

10. If **speed performance** issue:
    - Report OpenPose timing speed based on [this link](https://github.com/CMU-Perceptual-Computing-Lab/openpose/blob/master/doc/speed_up_openpose.md#profiling-speed).

**Comment by gineshidalgo99** [comment]
Good catch. Please, try with sAddDome = 0 for now. It will take me some time to actually collect those images. There are a lot in there.

**Comment by MasonSuWei** [comment]
Get it. Thanks for your response.

**Comment by MasonForFun** [comment]
Get it. Thanks for your response.

**Comment by jerry10301** [comment]
Hello,
I also need the data. Thank you a lot~

**Comment by zhongxiaofang-run** [comment]
Hello. I have meet a problem that can't be solved by myself. Please help me.
 **I want just train model  as origin model that output 18 points of body.**
First, I  have generated the COCO dataset LMDB files(without foot、hand、face dataset) by myself  followed your step descriped at  /Train/README.md.
I modified the  c_generateLmdbs.py  file   by uncommented line 136 and line 164， other code not modified.

Second, I have finished complied Caffe.  Runing script file  generateProtoTxt.py  generate the Caffe ProtoTxt .  I modified  generateProtoTxt.py follow  README.md,   set sAddFoot=0,sAddMpii=0, sAddFace=0 and sAddHands=0,sAddDome=0,sProbabilityOnlyBackground=0, sSuperModel=0,. 

my pose_trianing.prototxt is like this:(data layer)

layer {
  name: "image"
  type: "OPData"
  top: "image"
  top: "label"
  data_param {
    batch_size: 10
    backend: LMDB
  }
  op_transform_param {
    stride: 8
    max_rotate_degree: 45
    crop_size_x: 368
    crop_size_y: 368
    center_perterb_max: 40.0
    center_swap_prob: 0.0
    scale_prob: 1.0
    scale_min: 0.333333333
    scale_max: 1.5
    target_dist: 0.6
    number_max_occlusions: 2
    sigmas: "7.0"
    model: "COCO_17;COCO_17_17"
    source_background: "/mnt/zhong/openpose_train/dataset/lmdb_background"
    source: "/mnt/zhong/openpose_train/dataset/lmdb_coco"
    media_directory:""
    normalization: 0
   add_distance: false
  }
}


Error:  
I run the comand "bash train_pose.sh”.  Meet next error:
  terminate called after throwing an instance of 'std::runtime_error'
          what(): Error: mSources.size()!=nModels.size() at 56, OPDatalayer. src/caffe/openpose/layers/oPDatalayer.cpp
*** Aborted at 1581070034(unix time ) try "data -d @1581070034" if you are using GUN data ***
.....

Does it only train model which output 25 points (Body+ Foot)? Can't train model which output 18 points(Body) as origin openPose?

**Comment by zhongxiaofang-run** [comment]
Hello. I have meet a problem that can't be solved by myself. Please help me.
I want just train model as origin model that output 18 points of body.
First, I have generated the COCO dataset LMDB files(without foot、hand、face dataset) by myself followed your step descriped at /Train/README.md.
I modified the c_generateLmdbs.py file by uncommented line 136 and line 164， other code not modified.

Second, I have finished complied Caffe. Runing script file generateProtoTxt.py generate the Caffe ProtoTxt . I modified generateProtoTxt.py follow README.md, set sAddFoot=0,sAddMpii=0, sAddFace=0 and sAddHands=0,sAddDome=0,sProbabilityOnlyBackground=0, sSuperModel=0,.

my pose_trianing.prototxt is like this:(data layer)

layer {
name: "image"
type: "OPData"
top: "image"
top: "label"
data_param {
batch_size: 10
backend: LMDB
}
op_transform_param {
stride: 8
max_rotate_degree: 45
crop_size_x: 368
crop_size_y: 368
center_perterb_max: 40.0
center_swap_prob: 0.0
scale_prob: 1.0
scale_min: 0.333333333
scale_max: 1.5
target_dist: 0.6
number_max_occlusions: 2
sigmas: "7.0"
model: "COCO_17;COCO_17_17"
source_background: "/mnt/zhong/openpose_train/dataset/lmdb_background"
source: "/mnt/zhong/openpose_train/dataset/lmdb_coco"
media_directory:""
normalization: 0
add_distance: false
}
}

Error:
I run the comand "bash train_pose.sh”. Meet next error:
terminate called after throwing an instance of 'std::runtime_error'
what(): Error: mSources.size()!=nModels.size() at 56, OPDatalayer. src/caffe/openpose/layers/oPDatalayer.cpp
*** Aborted at 1581070034(unix time ) try "data -d @1581070034" if you are using GUN data ***
.....

Does it only train model which output 25 points (Body+ Foot)? Can't train model which output 18 points(Body) as origin openPose? @gineshidalgo99 @jerry10301 @MasonForFun
